multi_point.go

```diff
@@ -1,7 +1,9 @@
 package geoos
 
+import "github.com/spatial-go/geoos/algorithm/matrix"
+
 // A MultiPoint represents a set of points in the 2D Eucledian or Cartesian plane.
-type MultiPoint []Point
+type MultiPoint matrix.LineMatrix
```

Comment:
I think `MultiPoint` should be defined as []Point

Reply:
`MultiLineString` is defined as []LineString, `MultiPolygon` is defined as []Polygon

Reply:
take and done